flatpak/dev.neovide.Neovide.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.neovide.Neovide</id>
+  <name>Neovide</name>
+  <summary>Advanced graphical interface for Neovim</summary>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <developer id="dev.neovide">
+    <name>Kaylee Simmons</name>
+  </developer>
+  <description>
+    <p>Fully-featured Neovim GUI with some visual upgrades:</p>
+    <ul>
+      <li>Animated cursor</li>
+      <li>Smooth scrolling</li>
+      <li>Animated windows</li>
+      <li>Blurred floating windows</li>
+      <li>Emoji support</li>
+    </ul>
+  </description>
+  <categories>
+    <category>Utility</category>
+    <category>TextEditor</category>
+  </categories>
+  <launchable type="desktop-id">dev.neovide.Neovide.desktop</launchable>
+  <icon type="stock">dev.neovide.Neovide</icon>
+  <branding>
+    <color scheme_preference="light" type="primary">#79b242</color>
+    <color scheme_preference="dark" type="primary">#4d8a3b</color>
+  </branding>
+  <content_rating type="oars-1.1"></content_rating>
+  <url type="bugtracker">https://github.com/neovide/neovide/issues</url>
+  <url type="homepage">https://neovide.dev/index.html</url>
+  <url type="contact">https://discord.gg/SjFpZdQys6</url>
```

Comment:
can there be multiple contact urls? Might be worth including the matrix channel as well https://matrix.to/#/#neovide:matrix.org

Reply:
Unfortunately, adding a second url of type 'contact' causes a validation error.